cljfmt/src/cljfmt/main.clj

```diff
@@ -1,7 +1,7 @@
 (ns cljfmt.main
   "Functionality to apply formatting to a given project."
+  {:clj-kondo/config '{:linters {:unresolved-namespace {:exclude [cljfmt.diff]}}}}
```

Comment:
Is clj-kondo support necessary for bb support?

Reply:
Not necessary. I assume this is a rhetorical question meaning "please remove it"?

Reply:
It wasn't meant as rhetorical; I don't like to assume that something isn't part of the feature, as there have been times in the past where I've been wrong. But I do like to limit PRs to *just* the code necessary to implement the feature.